tools/internal_ci/helper_scripts/requirements.macos.txt

```diff
@@ -0,0 +1,7 @@
+cryptography==3.4.6
+PyJWT==2.0.1
+pyOpenSSL==20.0.1
+PyYAML==5.4.1
+requests==2.25.1
+scipy==1.5.4
```

Comment:
is scipy and tabulate actually needed on macOS?
I think you need those for bm_diff.py (which doesn't run on macos AFAIK) - can you please doublecheck?

Reply:
They aren't needed, I'll remove them. I was toying with the idea of using one superset of all python requirements for all tests to simplify the build file structure, but decided against it since it might be needlessly costly (~30mb extra scipy download for every test, for example).

Reply:
Now the list looks more reasonable, thanks!